skada/tests/test_reweight.py

```diff
@@ -52,3 +52,27 @@ def test_reweight_estimator(estimator, da_dataset):
     assert np.mean(y_pred == y_test) > 0.9
     score = estimator.score(X_test, y_test, sample_domain=sample_domain)
     assert score > 0.9
+
+
+def test_reweight_gamma_float(da_dataset):
+    X_train, y_train, sample_domain = da_dataset.pack_train(
+        as_sources=['s'],
+        as_targets=['t']
+    )
+
+    estimator = KLIEPAdapter(gamma=0.2)
```

Comment:
I thought we had a large test case for all adapters, maybe just add this as a additional case there?

Reply:
Yes sure